pkg/jobs/backoff.go

```diff
@@ -19,13 +26,17 @@ type backoff struct {
 }
 
 func (b *backoff) increaseTimeout() {
+	backoffMutex.Lock()
+	defer backoffMutex.Unlock()
```

Comment:
The lock can be added to *backoff

Reply:
I can try again. That's how I originally had intended. But, I had problems with the creation of the backoff object. It messes with values in the object, so I needed the lock before creating the object. Since backoff, at least currently, isn't messed with in code after creation (except for tests), I decided just making it global was ok.